### PR TITLE
If accessmode is PERF, set LIKWID_PERF_PID on startup

### LINKED ISSUE #45: On A64FX can't setup counter
cc: @giordano

On Ookami running `likwid-perfctr` works:

```
[vchuravy@fj-debug1 ~]$ likwid-perfctr -m  -C 0 -g FLOPS_DP julia perfctr.jl
--------------------------------------------------------------------------------
CPU name:	
CPU type:	Fujitsu A64FX
CPU clock:	0.00 GHz
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
Region saxpy, Group 1: FLOPS_DP
+-------------------+------------+
|    Region Info    | HWThread 0 |
+-------------------+------------+
| RDTSC Runtime [s] |   0.100502 |
|     call count    |          1 |
+-------------------+------------+

+----------------------+---------+------------+
|         Event        | Counter | HWThread 0 |
+----------------------+---------+------------+
|     INST_RETIRED     |   PMC0  |   95528990 |
|      CPU_CYCLES      |   PMC1  |  179848400 |
| FP_DP_FIXED_OPS_SPEC |   PMC3  |      20114 |
| FP_DP_SCALE_OPS_SPEC |   PMC4  |          0 |
+----------------------+---------+------------+

+--------------------------+------------+
|          Metric          | HWThread 0 |
+--------------------------+------------+
|    Runtime (RDTSC) [s]   |     0.1005 |
|        Clock [MHz]       |  1789.5007 |
|            CPI           |     1.8827 |
|     DP (FP) [MFLOP/s]    |     0.2001 |
| DP (FP+SVE128) [MFLOP/s] |     0.2001 |
| DP (FP+SVE256) [MFLOP/s] |     0.2001 |
| DP (FP+SVE512) [MFLOP/s] |     0.2001 |
+--------------------------+------------+

```
But using the CLI interface doesn't

```
julia> using LIKWID

julia> LIKWID.LibLikwid.likwid_pinThread(0)
1

julia> PerfMon.init(0)
true

julia> groupid = PerfMon.add_event_set("FLOPS_DP")
1

julia> PerfMon.setup_counters(groupid)
Setup of counters failed for thread 0
false
```

### PR DESCRIPTION
**Pull request by vchuravy**
Fixes #45

### CODE REVIEW

**Comment by vchuravy** [comment]
```
julia> using LIKWID
┌ Debug: /proc/sys/kernel/perf_event_paranoid is set to
│   perf_paranoid = 1
│   uid = 0x000000000a267b1b
└ @ LIKWID ~/.julia/packages/LIKWID/8lXWl/src/LIKWID.jl:91
┌ Debug: Setting environment variable LIKWID_PERF_PID
│   pid = 3926017
└ @ LIKWID ~/.julia/packages/LIKWID/8lXWl/src/LIKWID.jl:94

julia> LIKWID.LibLikwid.likwid_pinThread(0)
1

julia> PerfMon.init(0)
true

julia> groupid = PerfMon.add_event_set("FLOPS_DP")
1

julia> PerfMon.setup_counters(groupid)
true
```

File: src/LIKWID.jl [hunk 1]
```diff
@@ -75,10 +75,26 @@ import .GPUMarker: gpumarker, @gpumarker
 export GPUMarker, gpumarker, @gpumarker
 include("frequency.jl")
 
+function perf_event_paranoid()
+    open("/proc/sys/kernel/perf_event_paranoid", "r") do io
```
Comment (carstenbauer) [inline src/LIKWID.jl]: Does `/proc/sys/kernel/perf_event_paranoid` always exist and is it always readable by users? Is assume the answers are yes and yes. Just wondering if we need a default fallback here.

**Review by carstenbauer — COMMENTED** [review]
LGTM, I'll try to fix CI to see if tests still pass.

File: src/LIKWID.jl (hunk 1 above)
Comment (vchuravy) [reply src/LIKWID.jl]: Always on Linux yeah, let me throw in an ispath

**Pull request merged**